components/n-ui/ads/js/page-metrics.js

```diff
@@ -0,0 +1,76 @@
+const inMetricsSample = require('./utils').inMetricsSample;
```

Comment:
Just querying if we want to call this file "page-metrics", will we share this file with the slot-metrics also? or is this intended to be seperate from slot-level metrics?

Reply:
When we work on the slot-level metrics, we will have to extract some common methods into a separate file metrics-utils. I'd rather leave it like this for now and refactor/rename when we get to work on the slot metrics.